eservice/bin/register-with-ledger.sh

```diff
@@ -84,15 +84,21 @@ function Register {
         VAR_BASENAME=$(grep -o 'BASENAME:.*' ${eservice_enclave_info_file} | cut -f2- -d:)
 
         : "${PDO_LEDGER_URL:?Registration failed! PDO_LEDGER_URL environment variable not set}"
-        : "${PDO_LEDGER_KEY_SKF:?Registration failed! PDO_LEDGER_KEY_SKF environment variable not set}"
         : "PDO_IAS_KEY_PEM" "${PDO_IAS_KEY_PEM:?Registration failed! PDO_IAS_KEY_PEM environment variable not set}"
 
-        try ${SRCDIR}/sawtooth/bin/pdo-cli set-setting --keyfile $PDO_LEDGER_KEY_SKF --url $PDO_LEDGER_URL \
-            pdo.test.registry.measurements ${VAR_MRENCLAVE}
-        try ${SRCDIR}/sawtooth/bin/pdo-cli set-setting --keyfile $PDO_LEDGER_KEY_SKF --url $PDO_LEDGER_URL \
-            pdo.test.registry.basenames ${VAR_BASENAME}
-        try ${SRCDIR}/sawtooth/bin/pdo-cli set-setting --keyfile $PDO_LEDGER_KEY_SKF --url $PDO_LEDGER_URL \
-            pdo.test.registry.public_key "$(cat $PDO_IAS_KEY_PEM)"
+        if [ ${PDO_LEDGER_TYPE} == "ccf" ];
+	    then
+            try ${SRCDIR}/ccf_transaction_processor/scripts/register_enclave_attestation_verification_policy.py --logfile __screen__ --loglevel INFO \
+                --check_attestation --mrenclave ${VAR_MRENCLAVE} --basename  ${VAR_BASENAME} --ias-public-key "$(cat $PDO_IAS_KEY_PEM)"
```

Comment:
this is the simple way to do this. and it makes testing easy (just build the enclave and register it). but it does introduce a bunch of key management issues. how do i make this call from a host where the ccf tp is not running? somehow i have to grab the ledger member keys to invoke the operation. simple if everything is just running in the test environment. but how does this happen when it isn't?

Reply:
agree with all of your concerns, honestly, this is beyond the scope of this PR. should be taken up as part of a separate "deployment templates" PR. Current additions are made consistent with whatever exists for Sawtooth.